packages/prettier/index.js

```diff
@@ -0,0 +1,30 @@
+module.exports = {
+	arrowParens: 'always',
+	bracketSpacing: true,
+	jsxBracketSameLine: false,
+	jsxSingleQuote: false,
+	printWidth: 80,
+	quoteProps: 'as-needed',
+	semi: false,
```

Comment:
👀 ;

Reply:
yeah i think is not a good idea - will mean loads of codebases will need to reformat and it doesn't fix anything. will change it

Reply:
Aces 👍 I think everything else is a fair-enough non-default!